### PR TITLE
Add TSE Sage Embed

### PR DESCRIPTION
**Pull request by mouryab**

### CODE REVIEW

File: src/embed/sage.ts [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+     *
+     */
+    protected viewConfig: SageViewConfig;
+
+    private defaultHeight = 500;
+
+    // eslint-disable-next-line no-useless-constructor
+    constructor(domSelector: DOMSelector, viewConfig: SageViewConfig) {
+        super(domSelector, viewConfig);
+        if (this.viewConfig.fullHeight === true) {
```
Comment (girish-thoughtspot) [inline src/embed/sage.ts:97]: Full height support is mainly for LiveboardEmbed. Even in AppEmbed, this is specifically for when user embeds liveboard page in full App. I see we are not dispatching these actions in sage codebase, Is there a use case for full height in sageEmbed? If not we can remove this.

Comment (mouryab) [reply src/embed/sage.ts:97]: I will remove it.

Comment (mouryab) [reply src/embed/sage.ts]: @girish-thoughtspot I update it now

**Review by girish-thoughtspot — APPROVED** [review]

**Comment by ashubham** [comment]
1.

**Pull request closed without merging**